tests/test_sharpen.py

```diff
@@ -0,0 +1,16 @@
+from group_12 import sharpen
+import matplotlib.pyplot as plt
+
```

Comment:
You may want to add some tests corresponding to error handling tests in the function to check if your function handles invalid inputs correctly. Some short comments explaining what each test function is doing would also be great! The other tests look good.

Reply:
You are right. I will add few more tests for capturing errors.